src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -412,8 +412,12 @@ def _validate_vm_create_storage_profile(cmd, namespace, for_scale_set=False):
         else:
             raise CLIError('Unrecognized image type: {}'.format(image_type))
     else:
-        # did not specify image XOR attach-os-disk
-        raise CLIError('incorrect usage: --image IMAGE | --attach-os-disk DISK')
+        namespace.image = 'MicrosoftWindowsServer:WindowsServer:2022-datacenter-azure-edition:latest'
+        _parse_image_argument(cmd, namespace)
+        namespace.storage_profile = StorageProfile.ManagedPirImage
+        namespace.enable_secure_boot = True
+        namespace.enable_vtpm = True
+        namespace.security_type = 'TrustedLaunch'
```

Comment:
May I ask if these parameters are set to default value only in the absence of user input?
Hi @AjKundnani, could you please help confirm it?
```suggestion
        if namespace.enable_secure_boot is None:
            namespace.enable_secure_boot = True
        if namespace.enable_vtpm is None:
            namespace.enable_vtpm = True
        if namespace.security_type is None:
            namespace.security_type = 'TrustedLaunch'
```